tests/README.md

```diff
@@ -0,0 +1,77 @@
+## Backend development
+
+The backend test suite can be run locally. 
+
+### Start the services
+
+We need all but the main `backend` service running.  Since multiple `backend` instances are allowed, we will simply start everything up
+
+    # ENSURE THE IMAGES ARE CLEAN
+    docker-compose down
+    # SETUP ALL IMAGES
+    docker-compose up --build
+
+### Install into virtual environment
+
+Treeherder, Django, and the required libraries will ruin you local Python install. It is best to setup a virtual environment to contain the quirks it introduces.
+
```

Comment:
This isn't very clear to me - could you elaborate? (maybe I'm missing something crucial about python env and docker)

Reply:
Also ``ruin you Python install`` s/b ``ruin your``.

But I agree with Sarah.  Could you elaborate on "ruin" here?  :)  I presume you just mean that it will add a bunch of packages that you probably don't want to be global.  Using a ``venv`` keeps ``pip`` tidy, locally.

Reply:
I am guessing it is some installed binary compatibility problem:  After installing the production requirements, Django was not available.  Installing Django broke another project that used numpy.  I could only get things working by re-installing the Python (to clear the site_packages directory) so the pollution was gone.  `virtualenv` keeps the pollution, whatever it may be, contained.

I am unwilling to do that again, so let me change it to a less controversial line:

> Treeherder, requires specific library versions that will likely interfere with with what you have installed.